tests/beaker/tests/create_lv_with_param_alloc.rb

```diff
@@ -0,0 +1,82 @@
+require 'master_manipulator'
+require 'lvm_helper'
+require 'securerandom'
+
+test_name "FM-4579 - C96574 - create logical volume without parameter 'alloc'"
+
+#initilize
+pv = '/dev/sdc'
+vg = ("VolumeGroup_" + SecureRandom.hex(2))
+lv = [("LogicalVolume_" + SecureRandom.hex(3)), ("LogicalVolume_" + SecureRandom.hex(3)), \
```

Comment:
The first commit I had there appreviation: 'pv', 'vg', and 'lv'. In this PR I already changed them to 'PhysicalVolume_', ''VolumeGroup_", and "LogicalVolume_". I like to keep the securerandom part for being easier to maintain the codes if some errors happen in the future